app/assets/scripts/components/NavGlobalMenu.js

```diff
@@ -46,16 +46,16 @@ export default class NavGlobalMenu extends Component {
             <span>Relevant tools</span>
           </NavLink>
         </li>
-        {/* <li>
+        <li>
           <a
-            href='https://link-to-docs'
+            href='https://gep-user-guide.readthedocs.io'
             target='_blank'
```

Comment:
```suggestion
            target='_blank'
            rel='noreferrer noopener'
```

All external links should have `rel='noreferrer noopener'`
https://mathiasbynens.github.io/rel-noopener/